network-api/networkapi/wagtailpages/pagemodels/index.py

```diff
@@ -84,7 +84,7 @@ def get_context(self, request):
         return context
 
     def get_cache_key(self, locale):
-        return f"index_items_{self.slug}_{locale}"
+        return f"index_items_{self.slug}_{locale.language_code}"
```

Comment:
I updated this to use Locale Code (Ex: EN/FR/ES) because that is constant no matter what locale the user is visiting in. 

If we kept this as `locale`, then that could introduce confusion as the locale is returned in the language the user is visiting in. 

For example, English CMS users visiting `/es/blog` would get `Spanish` returned. However, Spanish users of the CMS would get `Espanol` returned.